src/cdc/runner.zig

```diff
@@ -204,6 +212,12 @@ pub const Runner = struct {
             .amqp_client = undefined,
         };
 
+        self.rate_limit = if (options.requests_per_second_limit) |limit|
+            // The rate limit is expressed in "requests per second":
+            RateLimit.init(time, .{ .limit = limit, .period = .seconds(1) })
+        else
+            null;
```

Comment:
Rather than increasing dimensionality with "if rate limit enabled else" branches, what do you think of using a default rate limit of `maxInt requests/second`, so that at least some of the rate limit code is always exercised?